### ISSUE TITLE
Resuming an interrupted charging-session does not work

### ISSUE THREAD

**Issue by Kunibert-007** [dannerph/homeassistant-keba#33]
When I interrupt a running charging-session with _keba_disable_ the charging-session stops correctly.
But when I try to resume it with the button _keba_enable_ there's no chance to start charging. The status in _keba_status_ is "authorization rejected".
In the Keba-App I see the hint "Unterbrochen (durch Auto)"/Interrupted by car and on the web-interface the state is "suspended".
To resume I can choose the option "Restart" in the web-interface, than the wallbox charges the car:
![Screenshot 2024-10-28 144615](https://github.com/user-attachments/assets/a940bacf-1c76-48c9-a4f0-59186c0fd7ba)

Has anybode the same issue? Or would it be possible to integrate the button for "Restart" in Home Assistant, so I could integrate it in the automation-flow for charging?

Keba P30 with Software v 3.10.57 (240521-093236)
Keba Charging Station Integration-Version 2.4.0
Home Assistant Core: 2024.10.4
Home Assistant Supervisor: 2024.10.3
Home Assistant Operating System: 13.2
Home Assistant Frontend: 20241002.4

**Comment by dannerph** [comment]
keba_disable also resets authorization, thus you need to authorize before resumingthe charging process. This can be done by the service call keba.start. As alternative use set_current 0 to pause the charging process

**Comment by andreasd** [comment]
i'm using two scripts to start and stop charging (triggered by ev smart charging integration or via buttons) and i see the same issue.

my start charging script includes the unlock and enable:

```
sequence:
  - action: lock.unlock
    metadata: {}
    data: {}
    target:
      entity_id: lock.keba_authentication
  - delay:
      hours: 0
      minutes: 0
      seconds: 5
  - action: button.press
    metadata: {}
    data: {}
    target:
      entity_id: button.keba_enable
```

the stop script just uses disables:

```
sequence:
  - action: button.press
    metadata: {}
    data: {}
    target:
      entity_id: button.keba_disable
```

i can use both scripts to enable and disable charging via buttons. when there is just a minute or so between stop and start, everything works well. but when more time passes, the enable does not work anymore. the auth fails even if using the unlock directly from the integration page (just does nothing). i can start charging via my p30's web interface and then also the integration interactions work again.

but as the smart charging integration sometimes pauses or restarts the charging, i can't manually restart the charging when this happens

**Comment by dannerph** [comment]
Could you please enable debug logging and provide those?

**Comment by andreasd** [comment]
thanks for getting back so quickly - yes, i'll do so and report back once it happens again or i can reproduce it.

**Comment by andreasd** [comment]
got it to reproduce by manually starting and stopping the sessions (using the two scripts posted above). i've attached the log file (keeping only entrys from the keba integration and the ev smart charging - if you need the full log, please let me know!)

[home-assistant_2025-01-21T08-24-44.279Z.log](https://github.com/user-attachments/files/18487228/home-assistant_2025-01-21T08-24-44.279Z.log)

basically what i did: started charging, letting it run for about 15 minutes, stopped charging and then after about 20 minutes tried starting it again. again, even if i go directly to the integration page and try to "unlock" the lock.keba_authentication, it does nothing.

thank you!
